cmd/kubeadm/app/master/apiclient.go

```diff
@@ -72,12 +72,9 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 		// TODO: use /healthz API instead of this
 		cs, err := client.ComponentStatuses().List(v1.ListOptions{})
 		if err != nil {
-			if apierrs.IsForbidden(err) {
-				fmt.Print("\r[apiclient] Waiting for the API server to create RBAC policies")
```

Comment:
printing a message in this case can be useful, but it shouldn't be RBAC-specific